### PR TITLE
aggregator: add version field.

### PR DESCRIPTION
**Pull request by niedbalski**
# Summary of this proposal

* Add Version field to aggregator struct(s).

Note that tests will not be added until this change lands and cloud gets re-synced.

## Notes for the reviewer

N/A.

## Issue(s) number(s)

*Disclaimer: Please do not create a Pull Request without creating an issue first.*

## Checklist for submitter

- [ ] My PR has a related issue/bug number.
- [ ] My PR provides tests
  - [ ] Integration tests are added/passes
  - [ ] Unit tests are added/passes
  - [ ] End-to-end tests added/passes
  - [ ] Static code check added/passes
  - [ ] Linting on documentation added/passes
  - [ ] Does not affect code coverage stats
- [ ] My PR requires updating dependencies
- [ ] My PR has the documentation changes required.

## Checklist for reviewer

- [ ] The proposal fixes a bug/issue or implements a new feature that is well described.
- [ ] The proposal has sufficient test cases that covers the changes.
  - [ ] If changes an API, it does not break backwards compatibility (-1 major version).
- [ ] If integration is required, the proposal has integration tests.
- [ ] The proposal does not break coverage stats
- [ ] The proposal has the required documentation changes.

## Backport

- [ ] Backport to the latest stable release.

### CODE REVIEW

File: types/aggregator.go [hunk 1]
```diff
@@ -2,11 +2,16 @@ package types
...
+const (
+	DefaultAggregatorVersion = "v0.1.12"
+)
+
 // Aggregator model.
 type Aggregator struct {
 	ID             string    `json:"id" yaml:"id"`
 	Token          string    `json:"token" yaml:"token"`
 	Name           string    `json:"name" yaml:"name"`
+	Version        string    `json:"version" yaml:"version"`
```
Comment (nicolasparada) [inline types/aggregator.go:14]: Is it required?

What will we do for the existing aggregators?
Fill a default value during migragion?

Comment (niedbalski) [reply types/aggregator.go:14]: yep, will default to DefaultAggregatorVersion

Comment (nicolasparada) [reply types/aggregator.go:14]: Can you update the spce then and add it to the `required` list then?

Comment (niedbalski) [reply types/aggregator.go:14]: done @nicolasparada

File: types/aggregator.go [hunk 2]
```diff
@@ -15,18 +20,21 @@ type Aggregator struct {
...
-	PublicRSAKey  []byte    `json:"publicRSAKey"`
-	Name          string    `json:"name"`
-	CreatedAt     time.Time `json:"createdAt"`
+	ID            string `json:"id"`
+	Token         string `json:"token"`
+	PrivateRSAKey []byte `json:"privateRSAKey"`
+	PublicRSAKey  []byte `json:"publicRSAKey"`
+	Name          string `json:"name"`
+	Version       string `json:"version"`
+
```
Comment (nicolasparada) [inline types/aggregator.go:36]: Why this extra space?

File: types/aggregator.go [hunk 3]
```diff
@@ -2,11 +2,16 @@ package types
 
 import "time"
 
+const (
+	DefaultAggregatorVersion = "v0.1.12"
```
Comment (nicolasparada) [inline types/aggregator.go:6]: I wonder somethimes if these defaults should be on this repo 🤔

**Review by nicolasparada — APPROVED** [review]
LGTM
Thanks :)

File: types/aggregator.go (hunk 3 above)
Comment (niedbalski) [reply types/aggregator.go:6]: @nicolasparada me too.

**Pull request merged**